src/develop/lightroom.c

```diff
@@ -907,6 +909,78 @@ static void _lrop(const dt_develop_t *dev, const xmlDocPtr doc, const int imgid,
       tcNode = tcNode->next;
     }
   }
+  else if(!xmlStrcmp(name, (const xmlChar *)"title"))
+  {
+	xmlNodePtr ttlNode = node;
+    while(ttlNode)
```

Comment:
sounds like there is indentation issues. please do not use tabulations. more issues like this below.